apps/palworld/1.1.0/docker-compose.yml

```diff
@@ -26,7 +26,7 @@ services:
       - RCON_ENABLED=${RCON_SWITCH}
       - RCON_PORT=${PANEL_APP_PORT_RCON}
       - QUERY_PORT=${PANEL_APP_PORT_QUERY}
-    image: thijsvanloef/palworld-server-docker:v1.1.0
+    image: thijsvanloef/palworld-server-docker:v1.2.1
     labels:  
       createdBy: "Apps"
 
```

Comment:
The difference is that the `image` value changes from **v1.1.0** to **v1.2.1**, indicating an update from version 1.1 to version 1.2 of the PalWorld server Docker image. It may be advisable to review if there have been any updates or changes in these new versions, such as bug fixes, improvements, or additional features.